jannovar-cli/src/main/java/de/charite/compbio/jannovar/cmd/annotate_pos/AnnotatePositionCommand.java

```diff
@@ -81,10 +82,10 @@ public void run() throws JannovarException {
 	}
 
 	private GenomeChange parseGenomeChange(String changeStr) throws JannovarException {
-		Pattern pat = Pattern.compile("(chr[0-9MXY]+):([0-9]+)([ACGTN]*)>([ACGTN]*)");
```

Comment:
ok. I thought it will be in vcf format, lite A AT or AT A

thanks for noticing!